libs/core/src/lib/flexible-column-layout/flexible-column-layout.component.ts

```diff
@@ -0,0 +1,421 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  ContentChild,
+  Component,
+  EventEmitter,
+  Input,
+  OnChanges,
+  OnDestroy,
+  OnInit,
+  Output,
+  SimpleChanges,
+  TemplateRef
+  } from '@angular/core';
+  import { fromEvent, Subscription } from 'rxjs';
+  import { debounceTime } from 'rxjs/operators';
+  import {
+  ONE_COLUMN_START_FULL_SCREEN,
+  ONE_COLUMN_MID_FULL_SCREEN,
+  ONE_COLUMN_END_FULL_SCREEN,
+  TWO_COLUMNS_START_EXPANDED,
+  TWO_COLUMNS_MID_EXPANDED,
+  TWO_COLUMNS_END_EXPANDED,
+  THREE_COLUMNS_MID_EXPANDED,
+  THREE_COLUMNS_END_EXPANDED,
+  THREE_COLUMNS_START_MINIMIZED,
+  THREE_COLUMNS_END_MINIMIZED,
+  SM_SCREEN_SIZE,
+  MD_SCREEN_SIZE,
+  LG_SCREEN_SIZE,
+  XL_SCREEN_SIZE,
+  ScreenSize,
+  Layout,
+  ColumnSeparatorValue
+} from './constants';
+  
+@Component({
+  selector: 'fd-flexible-column-layout',
+  templateUrl: './flexible-column-layout.component.html',
+  styleUrls: ['./flexible-column-layout.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class FlexibleColumnLayoutComponent implements AfterViewInit, OnChanges, OnDestroy, OnInit {
+
+  /**
+   * the template that provides the content of the first/start/left column
+   */
+  @ContentChild('startColumn')
+  startColumn: TemplateRef<any>;
+
+  /**
+   * the template that provides the content of the middle column
+   */
+  @ContentChild('midColumn')
+  midColumn: TemplateRef<any>;
+
+  /**
+   * the template that provides the content of the last/end/right column
+   */
+  @ContentChild('endColumn')
+  endColumn: TemplateRef<any>;
+  
+  /**
+   * the background design of the component
+   * Options include: 'solid' | 'translucent' | 'transparent'
+   * The default is set to 'solid'
+   */
+  @Input()
+  backgroundDesign: 'solid' | 'translucent' | 'transparent' = 'solid';
+
+  /**
+   * the layout of the component
+   * Options include: 'OneColumnStartFullScreen' | 'OneColumnMidFullScreen' |
+   * 'OneColumnEndFullScreen' | 'TwoColumnsStartExpanded' | 'TwoColumnsMidExpanded' |
+   * 'TwoColumnsEndExpanded' | 'ThreeColumnsMidExpanded' | 'ThreeColumnsEndExpanded' |
+   * 'ThreeColumnsStartMinimized' | 'ThreeColumnsEndMinimized'
+   */
+  @Input()
+  layout: Layout = ONE_COLUMN_START_FULL_SCREEN;
+
+  /**
+   * the event emitted on layout value cchange
+   */
+  @Output()
+  layoutChange: EventEmitter<Layout> = new EventEmitter<Layout>();
+
+  /**
+   * user defined break point for MD screens
+   * the default (Fiori 3) value is 599
+   */
+  @Input()
+  mdBreakPoint = 599;
```

Comment:
IMO it should follow breakpoints from documentation, so md starting from `961px`. 
The main reason why I want it to be brought is that every column have `320px` min-width. To display two columns in md in proper way, you need to have at least `960px`, to keep 67%/33% ratio

Reply:
![image](https://user-images.githubusercontent.com/26483208/100609652-a1ad9b00-330e-11eb-8ca9-370005b26ee6.png)

Reply:
I was using the break points specified in Fiori 3. Was not aware this component is using different break points. Changed to use the ones from the wiki for Flexible Column Layout